gbpservice/nfp/configurator/lib/fw_constants.py

```diff
@@ -13,9 +13,11 @@
 DRIVERS_DIR = 'gbpservice.nfp.configurator.drivers.firewall'
 SERVICE_TYPE = 'firewall'
 VYOS = 'vyos'
+ASAV = 'asav'
 NEUTRON = 'neutron'
 
 CONFIGURATION_SERVER_PORT = '8888'
+ASAV_CONFIGURATION_SERVER_PORT = '443'
```

Comment:
Yes. But, each vendor can have his own constants. So, this will go into the vendor dir.